### PR TITLE
add polyglot in server

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

**Comment by frankrousseau** [comment]
@clochix can you review that one?

File: .tx/config [hunk 1]
```diff
@@ -6,4 +6,3 @@ file_filter = client/app/locales/<lang>.json
 source_file = client/app/locales/en.json
```
Comment (clochix) [inline .tx/config]: Could you add the server files to the Transifex config ? Please.

File: server/controllers/applications.coffee [hunk 2]
```diff
@@ -438,14 +438,14 @@ module.exports =
...
-                    if err and err isnt "Not enough Memory"
+                    if err and err isnt localizationManager.t "not enough memory"
                         delete startedApplications[req.application.id]
                         return markBroken res, req.application, err
                     else if err
                         delete startedApplications[req.application.id]
                         data =
                             errormsg: err
-                            state: 'stopped'
+                            state: localizationManager.t 'stopped'
```
Comment (clochix) [inline server/controllers/applications.coffee]: Have you verified this state is a string displayed to the user and not an internal key ? Not sure

File: server/controllers/monitor.coffee [hunk 3]
```diff
@@ -1,4 +1,5 @@
-{MemoryManager} = require('../lib/memory')
+{MemoryManager}     = require('../lib/memory')
+localizationManager = require '../lib/localization_manager'
```
Comment (clochix) [inline server/controllers/monitor.coffee]: Why did you add the localizationManager to this file?

File: Cakefile [hunk 4]
```diff
@@ -85,7 +85,16 @@ buildJsInLocales = ->
         name     = file.replace '.json', '.js'
         fs.writeFileSync "./build/client/app/locales/#{name}", exported
         # add locales at the end of app.js
+    for file in fs.readdirSync './server/locales/'
```
Comment (aenario) [inline Cakefile]: In node, you can directly `locales = require('./somefile.json')` , why do you convert them to JS ?
